### PR TITLE
Create overview of communication tools and calendar processes and opt…

### PR DESCRIPTION
**Pull request by jmertic**
…ions for ASWF projects

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

**Review by lgritz — APPROVED** [review]
Thanks, this is really helpful!

File: process/collaboration_tools.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+**READ BEFORE SUBMITTING A REQUEST**
+
+Bots, tokens, and webhooks are reviewed on a case-by-case basis with most requests being rejected due to security, privacy, and usability concerns. Bots and the like tend to make a lot of noise in channels. The Academy Software Foundation Slack instance has thousands of members and it is crucial to ensure everyone has a great experience.
+
+Typically approved requests include: GitHub, tool requests, or other tools/platforms used to aid in the management of Slack itself. Please [submit a request] to have this reviewed.
+
+## Mailing lists
+
+Academy Software Foundation maintains a number of mailing lists for the foundation itself, it's members, and the community at large. These mailing lists are hosted at [groups.io](https://groups.io) and managed by the Linux Foundation staff and Academy Software Foundation community leadership.
```
Comment (jfpanisset) [inline process/collaboration_tools.md:51]: it's members -> its members

File: process/collaboration_tools.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+### Key mailing lists
+
+The primary mailing list for all techincal community discussion is [tac@lists.aswf.io](https://lists.aswf.io/g/tac). Each hosted project and working group has it's own mailing list as well, which are [listed in the project listing in the TAC repo](https://github.com/AcademySoftwareFoundation/tac#tac-projects).
```
Comment (jfpanisset) [inline process/collaboration_tools.md:55]: it's own -> its own

File: process/collaboration_tools.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+### Moderation and Moderation Guidelines
+
+By default, TSC voting members serve as the moderator for discussions in [tac@lists.aswf.io](https://lists.aswf.io/g/tac), and each hosted project's TSC will serve as moderator for it's mailing lists. Moderation privledges can be extended to others upon a vote by the TAC or respective TSC as applicable.
```
Comment (jfpanisset) [inline process/collaboration_tools.md:59]: it's mailing lists -> its mailing lists

File: process/collaboration_tools.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+- Understand the difference between a user abusing the resource or just having difficulty expressing comments and questions in English.
+- Be an example and role model to others in the community. In many cases, moderators are some of the first people new contributors will interact with.
+- Remember to check and recognize if you need take a break when you become frustrated or find yourself in a heated debate.
+- Help your colleagues if you recognize them in one of the stages of burnout.
+- Be helpful and have fun!
+
+### Mailing list best practices
+
+- Review your mailing list subscriptions regularly, and either change the delivery mechanism ( such as switching from indivudual emails to daily digests ) as it makes sense to keep the signal-to-noise level reasonable for you.
+- Keep your comments and conversation on topic
```
Comment (jfpanisset) [inline process/collaboration_tools.md:89]: End line item with period for consistency with other items?

File: process/collaboration_tools.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+- Recognize with a global community and other obligations, replies aren't instantaneous.
```
Comment (jfpanisset) [inline process/collaboration_tools.md:90]: missing "that" between "recognize" and "with"?

File: process/collaboration_tools.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+- Be sure to contact the list moderators if there are any [Code of Conduct] concerns.
+
+## Calendars
+
+You can stay subscribed to the meetings of the projects and committees of the Academy Software Foundation by subscribing to the coordinating mailing list. Each mailing list has a calendar which you can subscribe to you with your calendar client of choice.
+
+By default, the following permissions apply to calendars:
+
+- Voting TAC members can manage the [tac calendar].
+- TSC project chairpersons can manage thier respective project calendars.
```
Comment (jfpanisset) [inline process/collaboration_tools.md:101]: thier->their

File: process/collaboration_tools.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
+# Academy Software Foundation community collaboration tools
+
+Academy Software Foundation maintains a number of tools to enable collaboration amongst community participants. To help navigate these tools, this document provides a brief outline of how to use them and best practices.
+
+## Slack
```
Comment (jfpanisset) [inline process/collaboration_tools.md:5]: A couple of additional notes I would suggest:

- point out that some projects (OCIO notably) have their own Slack instance
- a reminder that Slack messages will expire, so by nature the medium is ephemeral: useful long term content should be transferred to a project repo, its documentation, or the Confluence wiki

File: process/collaboration_tools.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+- Visit the [Academy Software Foundation Calendar]
+- Click on the event you are updating and select Edit Event, scroll to the bottom of the Event and click on Delete Event.
+    - You'll need to select either Only This Event or All Meetings as applicable here as well.
+    
+The meeting update will be reflected in the meeting subscription with the next automatic update.
+
+### Calendar guidance
+
+To help individuals to not be inandated with calendar overload, try to follow these guidelines:
```
Comment (jfpanisset) [inline process/collaboration_tools.md:156]: inandated -> inundated

File: process/collaboration_tools.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+- The [tac calendar] should only be used for TAC meetings, meetings or events for the full OMP community, and kickoff meetings of new projects.
```
Comment (jfpanisset) [inline process/collaboration_tools.md:158]: OMP -> ASWF ?

File: process/collaboration_tools.md [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+Instructions for subscribing to an iCal calendar for popular calendaring tools:
+
+- [Google Calendar](https://support.google.com/calendar/answer/37100?co=GENIE.Platform%3DDesktop&oco=1) - note calendar updates only happen once daily
+- [Outlook](https://support.office.com/en-us/article/import-or-subscribe-to-a-calendar-in-outlook-com-cff1429c-5af6-41ec-a5b4-74f2c278e98c)
+- [Apple Calendar](https://support.apple.com/guide/calendar/subscribe-to-calendars-icl1022/mac)
+- [IBM/HCL Notes](https://www.ibm.com/support/knowledgecenter/SSKTWP_10.0.0/cal_add_cal_t.html)
+
+### Calendar management best practices
+
+- Do ensure that any meeting invites have clearly idenified web conference or location details.
```
Comment (jfpanisset) [inline process/collaboration_tools.md:174]: idenified -> identified

File: process/collaboration_tools.md [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+  - Unless the meeting is only in person, always provide a web conference for those who can't attend in person
+- Be sure the description of the meeting has an agenda or a link to the agenda of the meeting.
+- Since calendaring clients can not always be good at managing changes...
+  - If a meeting is to be rescheduled, delete the original meeting invite and add a new one with a title denoting it being rescheduled.
+  - Avoid cancelling meetings within 24 hours of the meeting
+  - For changes/cancellations, email the invitees as well to inform them of the cancellation
+  - When provisioning a new series of recurring meetings, instruct invites to remove old entries manually.
```
Comment (jfpanisset) [inline process/collaboration_tools.md:181]: invites -> invitees

**Review by jfpanisset — COMMENTED** [review]
Great stuff, should there also be a section in there about the Confluence wiki?

**Comment by jmertic** [comment]
Agreed @jfpanisset  - feel free to make the edits in there.

**Pull request merged**